client/src/main/java/io/pravega/client/admin/impl/StreamManagerImpl.java

```diff
@@ -184,14 +184,27 @@ public boolean checkStreamExists(String scopeName, String streamName) {
         return  Futures.getThrowingException(controller.checkStreamExists(scopeName, streamName));
     }
 
+    @Override
+    public boolean deleteScopeRecursive(String scopeName) {
+        NameUtils.validateUserScopeName(scopeName);
+        log.info("Deleting scope recursively: {}", scopeName);
+        return Futures.getThrowingException(controller.deleteScopeRecursive(scopeName));
+    }
+
     @Override
     public boolean deleteScope(String scopeName) {
         NameUtils.validateUserScopeName(scopeName);
         log.info("Deleting scope: {}", scopeName);
         return Futures.getThrowingException(controller.deleteScope(scopeName));
     }
 
+    /**
+     * A new API is created hence this is going to be deprecated.
+     *
+     * @deprecated As of Pravega release 0.10, replaced by {@link #deleteScopeRecursive(String)}.
```

Comment:
This API would be deprecated in Pravega version 0.11.0
I had provided this comment earlier as well and this was changed earlier, but now I see it going back to the wrong value again.

Reply:
Required update is complete now